api/catalog/api/utils/oauth2_helper.py

```diff
@@ -24,18 +24,22 @@ def get_token_info(token: str):
         token = AccessToken.objects.get(token=token)
     except AccessToken.DoesNotExist:
         return None, None, None
-    if token.expires >= dt.datetime.now(token.expires.tzinfo):
-        try:
-            application = models.ThrottledApplication.objects.get(accesstoken=token)
-            client_id = str(application.client_id)
-            rate_limit_model = application.rate_limit_model
-            verified = application.verified
-        except models.ThrottledApplication.DoesNotExist:
-            logger.warning("Failed to find application associated with access token.")
-            client_id = None
-            rate_limit_model = None
-            verified = None
-        return client_id, rate_limit_model, verified
-    else:
-        logger.warning("Rejected expired access token.")
+
+    try:
+        application = models.ThrottledApplication.objects.get(accesstoken=token)
+    except models.ThrottledApplication.DoesNotExist:
+        logger.warning("Failed to find application associated with access token.")
+
+    expired = token.expires < dt.datetime.now(token.expires.tzinfo)
+    if expired:
+        logger.info(
+            "rejected expired access token "
+            f"application.name={application.name} "
+            f"application.client_id={application.client_id} "
+        )
         return None, None, None
+
+    client_id = str(application.client_id)
+    rate_limit_model = application.rate_limit_model
```

Comment:
If the assignment of `application` on L29 fails, won't these lines fail as well? Did we want to re-raise after the except on L30 or return early?

Reply:
Oops, yes, I meant to return the `None, None, None` tuple.

Reply:
Fixed in https://github.com/WordPress/openverse-api/pull/792/commits/0e93aec3a2522b1f0e57eec1f6c22a449458b08c